### PR TITLE
Refactor : 2-D_MatrixSorting

### PR DESCRIPTION
**Pull request by anshmishra010**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

-> added comments above the line.
-> removed name written in description
-> added time and space complexity

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

pr link : https://github.com/TesseractCoding/NeoAlgo/pull/829

### CODE REVIEW

**Comment by anshmishra010** [comment]
@ankitaggarwal23 please see

File: C-Plus-Plus/cp/2-D_MatrixSorting.cpp [hunk 1]
```diff
@@ -1,6 +1,5 @@
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/2-D_MatrixSorting.cpp]: remove this line

File: C-Plus-Plus/cp/2-D_MatrixSorting.cpp [hunk 2]
```diff
@@ -13,12 +12,15 @@
...
+ * Enter the elements in an array : 
  * 5 4 7
  * 1 3 8
  * 2 9 6
  *
  * Output:
+ * Sorted 2-D Matrix is:
  * 1 2 3
  * 4 5 6
  * 7 8 9
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/2-D_MatrixSorting.cpp]: shift this part at the end of code

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by anshmishra010** [comment]
@ankitaggarwal23 done the requested changes

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**